Microsoft.Toolkit.Uwp.UI.Animations/Microsoft.Toolkit.Uwp.UI.Animations.csproj

```diff
@@ -15,22 +15,24 @@
       - AnimationExtensions: Blur, Fade, Light, Offset, Rotate, Saturation, Scale
     </Description>
     <PackageTags>UWP Toolkit Windows Animations Composition Connected Implicit XAML</PackageTags>
+    <LangVersion>9.0</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
   <!-- TODO: Figure out where these go... -->
   <ItemGroup>
     <Compile Remove="Behaviors\**" />
     <Compile Remove="Effects\**" />
-    <Compile Remove="Extensions\**" />
+    <Compile Remove="fg\**" />
```

Comment:
?

Reply:
I'm as confused as you are here 😄
Definitely just a typo, though really not sure how that happened in the first place ahahah
Will remove this when I finish the migration and remove all these leftover folders 🙂